### ISSUE TITLE
Waterline generates an invalid query (syntax error) for a model with only an ID

### ISSUE THREAD

**Issue by dylanpyle** [balderdashy/sails-postgresql#168]
I have a model that contains only an autoincrementing ID, for use as the engine of a ticket service ([something like this](http://code.flickr.net/2010/02/08/ticket-servers-distributed-unique-primary-keys-on-the-cheap/), but with multiple rows). The definition in Waterline is something like this:

```
{
  schema: true,
  autoPK: false,
  autoCreatedAt: false,
  autoUpdatedAt: false,
  associationFinders: false,
  dynamicFinders: false,
  attributes: {
    id: {
      autoIncrement: true,
      type: 'integer',
      unique: true,
      primaryKey: true
    }
  }
}
```

I've removed the `createdAt` and `updatedAt` fields, as I don't have any need for them given the purpose of the table Unfortunately, this means that when creating a model:

````
MyModels.create().then ...
```

Waterline generates an invalid statement, and throws an error:

```
ERROR:  syntax error at or near ")" at character 25
STATEMENT:  INSERT INTO "mymodels" () values () RETURNING *
```

Using waterline 0.10.13, sails-postgresql 0.10.14

**Comment by devinivy** [comment]
Thanks for the heads-up.  I think this issue most likely lives in waterline-sequel: https://github.com/balderdashy/waterline-sequel/blob/d1f1b51c5f6ffdb118275250994607dba140d506/sequel/index.js#L163-L169 .  The auto-incrementing primary key should be inserted as `NULL` if everything else is empty.

In the meantime, you may be able to sidestep this by providing `{id: null}` to `Collection.create`.

**Comment by sailsbot** [comment]
Thanks for posting, @dylanpyle.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
